### ISSUE TITLE
[a11y-defect-2]: Edit & Remove buttons do not have distinct aria-labels

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#71635]
### Point of contact

@jeana-adhoc

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

On the [disability exam page](https://staging.va.gov/records/request-personal-records-form-20-10206/disability-exam-details), when there are multiple edit and remove buttons on the page, aria-labels must be added to the buttons and be made unique so that you know what is being edited.

### Reproduction steps

1. Enable a screen reader
2. Navigate to the [disability exam page](https://staging.va.gov/records/request-personal-records-form-20-10206/disability-exam-details)
3. Add 2 or more exam dates
4. Go back and click "Edit" You will hear "Edit exam date" for each card.

### Proposed solution or next steps

The aria-label for each edit button should be "Edit exam date for `actual exam date`" And include the actual date that is being edited.

The aria-label for each remove button should be "Remove exam date for `actual exam date`" And include the actual date that is being edited.

### References, articles, or WCAG support

1. [WCAG2.1 Understanding Success Criterion 2.5.3: Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html)

### Type of issue

- [ ] Axe-core
- [X] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [X] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by tlei123** [comment]
[PR](https://github.com/department-of-veterans-affairs/vets-website/pull/27114) opened.

**Comment by tlei123** [comment]
[PR](https://github.com/department-of-veterans-affairs/vets-website/pull/27114) merged &mdash; changes should be in Staging by ~4:45 PM ET.

**Comment by tlei123** [comment]
@tbaker1026 @jeana-adhoc @rachelshearerux , fix is now [in Staging](https://staging.va.gov/records/request-personal-records-form-20-10206/introduction), ready for validation.

**Comment by jeana-adhoc** [comment]
This works great!! Verified in staging. Edit and remove buttons aria-labels include the date making them unique.
